meteorite_fall.js

```diff
@@ -51,6 +62,12 @@ function initializeMeteorites() {
 
 // key events
 document.addEventListener('keydown', (e) => {
+    if (e.key == "p")
```

Comment:
Key "p" is not as easy to press as the spacebar would be. Perhaps we could change this to "Spacebar" instead of "p"?

Reply:
ok boss

Reply:
love you

Reply:
done